src/management/projects.ts

```diff
@@ -256,10 +346,34 @@ export async function getProjectPage(
     );
   }
 
-  return {
-    projects: apiResponse.body.results || [],
-    nextPageToken: apiResponse.body.nextPageToken,
-  };
+  return projectPage;
+}
+
+/**
+ * Lists a page of available Google Cloud Platform projects that are available to have Firebase
+ * resources added, using the paginated API, identified by the page token and its size.
+ */
+export async function getAvailableCloudProjectPage(
+  pageSize: number = PROJECT_LIST_PAGE_SIZE,
+  pageToken?: string
+): Promise<ProjectPage<CloudProjectInfo>> {
+  let projectPage;
+
+  try {
+    projectPage = await getProjectPage<CloudProjectInfo>("/v1beta1/availableProjects", {
+      responseKey: "projectInfo",
+      pageSize,
+      pageToken,
+    });
+  } catch (err) {
+    logger.debug(err.message);
+    throw new FirebaseError(
+      "Failed to list available Google Cloud Platform projects. See firebase-debug.log for more info.",
+      { exit: 2, original: err }
+    );
+  }
+
+  return projectPage;
```

Comment:
The value above (`await getProjectPage`) can be returned - the variable `projectPage` here is unnecessary.

Reply:
Done